spring/spring-data/src/test/java/io/quarkus/ts/spring/data/primitivetypes/BookResourceIT.java

```diff
@@ -78,6 +79,19 @@ void testJpaFieldsMapping() {
         assertThat(book.getComments(), equalTo(commentUpdated));
     }
 
+    @Test //Regression test for https://github.com/quarkusio/quarkus/pull/20568
```

Comment:
```suggestion
    @Tag("QUARKUS-1521")
    @Test
```
A tag links it to JIRA, where is also the GH link.